examples/LaravelSampleApp/app/Http/Controllers/GoogleAdsApiController.php

```diff
@@ -178,6 +176,9 @@ public function showReportAction(
             // Converts each result as a Plain Old PHP Object (POPO) using JSON.
             $results[] = json_decode($googleAdsRow->serializeToJsonString(), true);
         }
+        // We need to slice the array here so it shows the correct set of results based on the
```

Comment:
It would be more efficient to do this filtering in the `foreach` loop above.

Reply:
Good point. Let me modify it and get back to you.